### DISCUSSION TITLE
Disable Auto-Open submenus on Mouse Over

### DISCUSSION THREAD

**Discussion by zoltanpapp82** [Open-Shell/Open-Shell-Menu#1197] [Q&A] [answered]
Dear Community Members,

I always disabled the auto menu opening in all Windows programs. It does not have any benefit from my perspective... if I stop with the cursor on an entry, which has submenu it opens, with no reason (to me):
![OpenShell Mouse Over](https://user-images.githubusercontent.com/117342815/199667555-e7fc71fb-7af6-4bf1-acd9-5de9e96b2eb9.png)

I wanted to disable in OpenShell as well, but does not find any possibility to disable.

Could you tell if it is possible? It was possible to get it disabled in Win7: "Open submenus when I pause on them with the mouse pointer", and fortunately disable in Win10 by default.

Thank you,
Zoltan

**Comment by ge0rdi** [comment, marked as answer]
Hello.

You can change `Settings` -> `General Behavior` -> `Menu delay` to some large value (like 40000 or so).
After that menus will effectively not open automatically.

**Reply by zoltanpapp82** [reply]
Dear ge0rdi,

thank you for the prompt response! Now it is working like W7.

Best Regards,
Zoltan
